### ISSUE TITLE
5 issues I found in one round involving new medbay's organ transplant thing

### ISSUE THREAD

**Issue by xevonix** [vgstation-coders/vgstation13#1655]
1: The organ printing machine would print an organ, then before I could pick it up it would be gone. I even ALT-Clicked the tile to see if it was under it, but nope. It was just gone.

2: When trying to pick-up on a surgery someone else had started, it would just attack them with the item instead of continuing the surgery. The person who initiated the surgery could also not continue after that as well.

3: When doing certain surgeries, such as in my case I was attempting to de-brain a patient, after using the scalpel to start the surgery it would tell me "You can't see any useful way to use the circular saw on _______". This was an issue that was prevalent during the short beta testing session I had with it.

4: When attempting a liver-removal surgery, after severing the liver from the body I went in with the Hemostat to remove it from the cavity only to begin picking chips out of the person's brain somehow. Not sure why these two seem to happen. I believe I also experienced this same bug a few days ago when doing a lung transplant.

5: After defibbing a patient and taking him to surgery to fix his liver, I became unable to replace it thanks to the bug which the organs printed disappear. So I went to do a mercy killing so I could just clone them, and I was met with another issue that I had happen a few days ago as well. When I attempted to end their life, not only would it refuse to use certain medical tools during the process, but it also stuck him in some sort of state that prevented me from killing him. I could inflict as much damage as I wanted on the guy, but it would not finally kill him. Nor was I able to remove his head due to the surgery bug. We fed this guy 30 units of Creatine, one 50 unit water pill, one 50 unit potassium pill, beat him with a crowbar in the head and choked him for around 2 minutes. Still he did nothing but stay in a black bar of health, but still gasp and have a health HUD indicator that he was still alive. I had to wait until a borg was able to come to medbay and remove his head.
TL;DR, couldn't mercy-kill a incurable patient due to strange reoccurring glitch.

3/5 of these glitches made it impossible to play new medbay. Please fix them.
